fbw-common/src/wasm/systems/systems/src/navigation/adirs.rs

```diff
@@ -534,11 +555,23 @@ impl AirDataInertialReferenceUnit {
         context: &UpdateContext,
         overhead: &AirDataInertialReferenceSystemOverheadPanel,
         align_time: AlignTime,
+        aircraft_preset_quick_mode: bool,
         simulator_data: AdirsSimulatorData,
     ) {
-        self.adr.update(context, overhead, simulator_data);
-        self.ir
-            .update(context, &self.adr, overhead, align_time, simulator_data);
+        self.adr.update(
+            context,
+            overhead,
+            aircraft_preset_quick_mode,
+            simulator_data,
+        );
+        self.ir.update(
+            context,
+            &self.adr,
+            overhead,
+            align_time,
+            aircraft_preset_quick_mode,
+            simulator_data,
+        );
```

Comment:
Perhaps add a note here that this is to be refactored (abstracted) away from the `AirInertialReferenceUnit` struct, I don't there is a need for this struct to require awareness of ``aircraft_preset_quick_mode`` at this level. 

However solving this would require building a higher level wrapper and driver which would pass a mutable reference e.g. ``&mut adirs`` as a parameter in its update function, and perform manipulation like this, which is maybe a deep refactor and re-architecture that may be beyond the scope of this PR.

Reply:
I will add a note - I don't want to mess too much with the rust stuff.